src/_guides/language/sound-dart.md

```diff
@@ -0,0 +1,696 @@
+---
+layout: guide
+title: "Sound Dart Guide"
+description: "."
+---
+
+This guide tells you why and how to write sound (type safe) Dart code.
+You'll learn how to use strong mode to enable soundness, as well as
+how to substitute types safely.
+
+{% comment %}
+Not ready yet...
+If you are already using strong mode Dart, you might also check out
+[Sound Dart: Common Problems](/guides/language/sound-dart-common-problems).
+{% endcomment %}
+
+<aside class="alert alert-info" markdown="1">
+**Note:** The terms "sound Dart", "strong mode Dart", and "type safe Dart"
+are sometimes used interchangeably. _Strong mode_ is Dart's implementation
+of a sound type system. With strong mode enabled,
+Dart is a type safe language. "Classic Dart" refers to Dart
+before soundness was added to the language.
+
+Eventually, soundness won't be optional in the Dart language, and strong mode
```

Comment:
I don't think we've officially announced the "won't be optional part of this yet".